pkg/meta/interface.go

```diff
@@ -102,8 +105,24 @@ const (
 
 const MaxName = 255
 const MaxSymlink = 4096
+
+type Ino uint64
```

Comment:
not necessary

Reply:
origin code, just move to the same place